packages/frames/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@poap-xyz/frames",
+  "version": "0.0.1",
```

Comment:
I think this is meant to match the others packages versions, I let @rlajous comment on that.

Reply:
As this is more than a helper package, if it works well, we can put it with version `1.0.0`. Also no other package will be dependant of this.